README.md

```diff
@@ -1,13 +1,51 @@
-# KituraContracts
+<p align="center">
+<a href="http://kitura.io/">
+<img src="https://raw.githubusercontent.com/IBM-Swift/Kitura/master/Sources/Kitura/resources/kitura-bird.svg?sanitize=true" height="100" alt="Kitura">
+</a>
+</p>
+
 
-[![Build Status - Master](https://travis-ci.org/IBM-Swift/KituraContracts.svg?branch=master)](https://travis-ci.org/IBM-Swift/KituraContracts)
-![macOS](https://img.shields.io/badge/os-macOS-green.svg?style=flat)
-![Linux](https://img.shields.io/badge/os-linux-green.svg?style=flat)
+<p align="center">
+<a href="http://www.kitura.io/">
+<img src="https://img.shields.io/badge/docs-kitura.io-1FBCE4.svg" alt="Docs">
+</a>
+<a href="https://travis-ci.org/IBM-Swift/Kitura-net">
+<img src="https://travis-ci.org/IBM-Swift/KituraContracts.svg?branch=master" alt="Build Status - Master">
+</a>
+<img src="https://img.shields.io/badge/os-Mac%20OS%20X-green.svg?style=flat" alt="Mac OS X">
```

Comment:
This line needs to reflect the rename of Mac OS X to macOS. 
`<img src="https://img.shields.io/badge/os-macOS-green.svg?style=flat" alt="macOS">`